src/ui/qgsnewhttpconnectionbase.ui

```diff
@@ -201,10 +208,29 @@
            </property>
           </widget>
          </item>
-         <item row="4" column="0" colspan="2">
-          <widget class="QCheckBox" name="cbxWmsIgnoreReportedLayerExtents">
+         <item row="5" column="0" colspan="2">
+          <widget class="QCheckBox" name="cbxIgnoreGetMapURI">
            <property name="text">
-            <string>Ignore reported layer extents</string>
+            <string>Ignore GetMap/GetTile/GetLegendGraphic URI reported in capabilities</string>
+           </property>
+          </widget>
+         </item>
+         <item row="2" column="1">
+          <widget class="QSpinBox" name="sbFeatureCount">
+           <property name="toolTip">
+            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Specify a default value for FEATURE_COUNT when a new layer is created from this connection. &lt;/p&gt;&lt;p&gt;FEATURE_COUNT defines the maximum number of results returned by a GetFeatureInfo request, if not specified the server default value (usually 1) will be used.&lt;/p&gt;&lt;p&gt;Set to 0 to use server default: no FEATURE_COUNT parameter will be added to the request.&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```

Comment:
@elpaso if I am correct, with a default value set in that spinbox, this option can't be unspecified. IOW would it be correct to label the tooltip as below?

_Specify a default value for FEATURE_COUNT when a new layer is ~~created~~loaded from this connection.
FEATURE_COUNT defines the maximum number of results returned by a GetFeatureInfo request.~~, if not specified the server default value (usually 1) will be used.~~
Set to 0 to use server default (usually 1): no FEATURE_COUNT parameter will be added to the request._
PS: I didn't check the behavior in QGIS. Comment only based on PR discussion.